packages/carbon-graphs/src/js/helpers/constructUtils.js

```diff
@@ -7,36 +7,19 @@ import utils from './utils';
 
 /**
  * Simple utility that checks if the input is an array or objects or simple object.
- * Iterate over the array and calls the loader callback function.
+ * Iterate over the array and calls the task callback function.
  *
  * @private
  * @param {object|Array} input - Input can be a single object or an array of objects
- * @param {Function} loader - loader callback function that will be called with the input as an argument
+ * @param {Function} task - task callback function that will be called with the input as an argument
  * @returns {undefined} returns nothing
  */
-const contentLoadHandler = (input, loader) => {
-  if (utils.isArray(input)) {
-    input.forEach((i) => loader(i));
+const contentHandler = (input, task) => {
+  if(utils.isArray(input)) {
+    input.forEach((i) => task(i));
   } else {
-    loader(input);
+    task(input);
   }
 };
 
-/**
- * Simple utility that checks if the input is an array or objects or simple object.
- * Iterate over the array and calls the unloader callback function.
- *
- * @private
- * @param {object|Array} input - Input can be a single object or an array of objects
- * @param {Function} unloader - unloader callback function that will be called with the input as an argument
- * @returns {undefined} returns nothing
- */
-const contentUnloadHandler = (input, unloader) => {
-  if (utils.isArray(input)) {
-    input.forEach((i) => unloader(i));
-  } else {
-    unloader(input);
-  }
-};
-
-export { contentLoadHandler, contentUnloadHandler };
+export { contentHandler};
```

Comment:
nit: space before the second curly brace

Reply:
[fixed here](https://github.com/cerner/terra-graphs/pull/29/commits/9757af4b4ac775c715ed6c091743561d8a9ba5d0)